app/Http/Controllers/LandingPagesController.php

```diff
@@ -42,6 +42,44 @@ public function show($id, $slug = null)
         if ($canonicalRedirect = $this->getCanonicalRedirect($canonicalPath)) {
             return $canonicalRedirect;
         }
+
+        if (!$item->is_published) {
+            $this->seo->nofollow = true;
+            $this->seo->noindex = true;
+        }
+
+        $types = collect(LandingPage::TYPES);
+        switch ($item->type_id) {
+            case $types->search('Home'):
+                $this->seo->setTitle($item->meta_title ?: $item->title ?: "Downtown Chicago's #1 Museum");
+                $this->seo->setDescription($item->meta_description ?: "Located downtown by Millennium Park, this top art museum is TripAdvisor's #1 Chicago attraction—a must when visiting the city.");
+                break;
+
+            case $types->search('Visit'):
+                $this->seo->setTitle($item->meta_title ?: $item->title ?: 'Visit a Chicago Landmark');
+                $this->seo->setDescription($item->meta_description ?: 'Looking for things to do in Downtown Chicago? Plan your visit, find admission pricing, hours, directions, parking & more!');
+                $this->seo->setImage($item->imageFront('hero') ?? $item->imageFront('visit_mobile'));
+                break;
+
+            case $types->search('Research and Resources'):
+                $this->seo->setTitle($item->meta_title ?: $item->title ?: 'Research & Resources');
+                $this->seo->setDescription($item->resources_landing_intro);
+                $this->seo->setImage($item->imageFront('research_landing_image'));
+                break;
+
+            case $types->search('My Museum Tour'):
+                $this->seo->setTitle($item->meta_title ?: $item->title);
+                $this->seo->setDescription($item->meta_description);
+                $this->seo->setImage($item->imageFront('header_my_museum_tour_header_image') ?? $item->imageFront('header_my_museum_tour_header_image_mobile'));
+                break;
```

Comment:
Why is the MMT SEO data in two places, here and in the MMT controller?

Reply:
This is for the landing page, and the MMT controller is for individual tours.